includes/Modules/Analytics_4.php

```diff
@@ -226,9 +226,15 @@ protected function create_data_request( Data_Request $data ) {
 					);
 				}
 
+				$timezone = get_option( 'timezone_string' );
+				if ( empty( $timezone ) ) {
+					$timezone = 'UTC';
+				}
+
 				$property = new Google_Service_GoogleAnalyticsAdmin_GoogleAnalyticsAdminV1alphaProperty();
 				$property->setParent( self::normalize_account_id( $data['accountID'] ) );
 				$property->setDisplayName( wp_parse_url( $this->context->get_reference_site_url(), PHP_URL_HOST ) );
+				$property->setTimeZone( $timezone );
```

Comment:
Is setting a timezone here necessary? I don't think we have been doing that when creating UA properties. I see why it might be nice, but it also introduces complexity and might not be as straightforward as it is (since WordPress probably supports a different list of timezone strings than Google Analytics does).

Does not having a timezone here produce an error? If it doesn't and the field is optional, we should continue to not set it instead.

Reply:
Yes, it is required now. The createProperty request fails without a timezone.

https://developers.google.com/analytics/devguides/config/admin/v1/rest/v1alpha/properties#Property
![developers google com_analytics_devguides_config_admin_v1_rest_v1alpha_properties](https://user-images.githubusercontent.com/873049/120848741-d7616400-c57d-11eb-8523-39a84f38161b.png)